services/core-web/src/styles/components/Tables.scss

```diff
@@ -99,7 +99,7 @@ tr.ant-table-expanded-row, tr.ant-table-expanded-row:hover {
   }
 }
 
-.wrapped-text {
+.numberO .wrapped-text {
```

Comment:
There's a few different use cases for the .wrapped-text class, are you sure they're all within the .number0 class? Is the number0 class like the antd class for indicating the row is level 0 or what is its use case?

Reply:
`.number0` was removed from this, it is back to the way it is in developer branch